File: cifrado/web/codigo/Python-Nagare/Counter/Counter/gestor.py

from cesar import Cesar
from reverse import Reverse
from reverseG import ReverseG


class Gestor:
    def __init__(self):
        self.cesar = Cesar()
        self.reverse = Reverse()
        self.reverse_g = ReverseG()
        self.pala = "";

    def cifrar(self):
        self.cesar.palabra = self.pala
        self.reverse.palabra = self.pala
        self.reverse_g.palabra = self.pala

        self.cesar.cifrar()
        self.reverse.cifrar()
        self.reverse_g.cifrar()

    def cifrarSinActualizacion(self):
        self.cesar.cifrar()
        self.reverse.cifrar()
        self.reverse_g.cifrar()

File: cifrado/web/codigo/CSharp-ASP/CifradosWebASP/CifradosWebASP/App_Code/Gestor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class Gestor
{
    public Cesar cesar;
    public Reverse reverse;
    public ReverseG reverseG;


    public Gestor()
    {
        cesar = new Cesar();
        reverse = new Reverse();
        reverseG = new ReverseG();
    }
    public void cifrar(String pala)
    {
        cesar.palabra = pala;
        reverse.palabra = pala;
        reverseG.palabra = pala;

        cesar.Cifrar();
        reverse.Cifrar();
        reverseG.Cifrar();
    }
    public void cifrarSinActualizacion()
    {
        cesar.Cifrar();
        reverse.Cifrar();
        reverseG.Cifrar();
    }
}